pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -282,8 +292,19 @@ func (manager *Manager) reconcileCluster(cluster *clusterv1alpha2.PediaCluster)
 	}
 
 	manager.UpdateClusterSynchroCondition(cluster.Name, clusterv1alpha2.RunningConditionReason, "", metav1.ConditionTrue)
+	syncResources := cluster.Spec.SyncResources
+	if refName := cluster.Spec.SyncResourcesRefName; refName != "" {
+		ref, err := manager.clusterSyncResourcesLister.Get(refName)
+		if err != nil && !apierrors.IsNotFound(err) {
+			// TODO: Use more obvious method to let users know.
+			klog.Warning(err, "Failed to get SyncResourcesRef of cluster", cluster.Name)
+		}
+		if ref != nil {
+			syncResources = append(syncResources, ref.Spec.SyncResources...)
+		}
```

Comment:
if err != nil && !apierrors.IsNotFound(err)，we should return err

if (err == nil && apierrors.IsNofFound(err)) || ref == nil, we should warning.